src/main/java/edu/unc/lib/boxc/migration/cdm/model/MigrationProject.java

```diff
@@ -36,6 +36,7 @@
     public static final String GROUP_MAPPING_FILENAME = "group_mappings.csv";
     public static final String SIPS_DIRNAME = "sips";
     public static final String REDIRECT_MAPPING_FILENAME = "redirect_mappings.csv";
+    public static final String REDIRECT_MAPPING_INDEX_FILENAME = "redirect_mapping_index.db";
```

Comment:
Since this filepath is only going to be used for tests, I would prefer this constant and the path construction be moved into a test class. You could move them into SipServiceHelper.

If you do this, then you will need to add a way to pass the database connection string into RedirectMappingIndexService, which you could do by adding a setter (`setConnectionString`) to the service and then having the test call the setter and pass in value`"jdbc:sqlite:" + testHelper.getRedirectMappingIndexPath()`

You'll need to do something like this eventually either way since there will be two database types, so if you'd like we could defer this to the next ticket.